<ui version="4.0" >
 <class>LogitechMouseBase</class>
 <widget class="QWidget" name="LogitechMouseBase" >
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="cordlessNameLabel" >
     <property name="text" >
      <string>Cordless Name</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="resolutionSelector" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>Sensor Resolution</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QRadioButton" name="button400cpi" >
        <property name="text" >
         <string>400 counts per inch</string>
        </property>
        <property name="buttonGroupId" stdset="0" >
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="button800cpi" >
        <property name="text" >
         <string>800 counts per inch</string>
        </property>
        <property name="buttonGroupId" stdset="0" >
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="batteryBox" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>Battery Level</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QProgressBar" name="batteryBar" >
        <property name="maximum" >
         <number>7</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="channelSelector" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>RF Channel</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QRadioButton" name="channel1" >
        <property name="text" >
         <string>Channel 1</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
        <property name="buttonGroupId" stdset="0" >
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="channel2" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Channel 2</string>
        </property>
        <property name="buttonGroupId" stdset="0" >
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="permissionProblemText" >
     <property name="text" >
      <string>You have a Logitech Mouse connected, and libusb was found at compile time, but it was not possible to access this mouse. This is probably caused by a permissions problem - you should consult the manual on how to fix this.</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
  <resources/>
 <connections/>
</ui>
